AIDriver.lua

```diff
@@ -493,6 +500,49 @@ function AIDriver:driveCourse(dt)
 	end
 end
 
+---currentlyInTrigger is related to waypoints, self.course:hasTriggerWithinDistance()
+---self.triggerHandler:isInTrigger() is old code from somewhere that's still needed
+function AIDriver:isInTrigger()
+	--temp for testing
+--	local newIx = self.ppc:getRelevantWaypointIx()
+--	local backOffset = self:getBackMarkerOffset(self.vehicle) + 2
+--	self.course:hasTriggerWithinDistanceRaycasts(newIx,20,backOffset,self,"hasTriggerRaycastCallback")
+	
+	return (self.currentlyInTrigger or self.triggerHandler:isInTrigger()) and self.bunkerSiloManager == nil or false
+end
+
+function AIDriver:setIsInTrigger()
+	self.currentlyInTrigger = true
+end
+
+
+
```

Comment:
Course:hasTriggerRaycast(startIx,driverClass,driverCallback)  callback,
sadly it's not working as intended, as raycast only seem to detect node objects, but not a trigger bounding box of for example a fueltrigger of a fuel refill trailer.
=> so it's not used right now